xbmc/guilib/GUIFontCache.h

```diff
@@ -231,10 +231,10 @@ struct CGUIFontCacheDynamicPosition
 
 struct CVertexBuffer
 {
-  void *bufferHandle;
+  unsigned int bufferHandle; // this is really a GLuint
```

Comment:
this breaks Win32, so needs sync in GUIFontTTFDX.cpp
>(ClCompile target) -> 
         ..\..\xbmc\guilib\GUIFontTTFDX.cpp(194): error C2665: 'CVertexBuffer::CVertexBuffer' : none of the 3 overloads could convert all the argument types [c:\jenkins\slave\workspace\WIN-32\project\VS2010Express\XBMC.vcxproj]
         ..\..\xbmc\guilib\GUIFontTTFDX.cpp(204): error C2446: '!=' : no conversion from 'unsigned int' to 'nullptr' [c:\jenkins\slave\workspace\WIN-32\project\VS2010Express\XBMC.vcxproj]